doc/whats_new/upcoming_changes/sklearn.inspection/32651.fix.rst

```diff
@@ -0,0 +1,3 @@
+- :class:`~sklearn.inspection.permutation_importance` now supports
+  Polars DataFrames when used in pipelines with :class:`~sklearn.compose.ColumnTransformer`
+  and string column names. By :user:`Shardul369-coder <Shardul369-coder>`
```

Comment:
I don't know if we should consider this a fix or an "enhancement". Feel free to rename the changelog entry filename accordingly if you prefer to mark it as an "enhancement" rather than a "fix".

Reply:
Actually the fix is primarily about the support of polars dataframes in the permutation_importance function, so let's update this entry to reflect that.